src/main/resources/assets/computercraft/lua/rom/programs/http/wget.lua

```diff
@@ -40,10 +36,20 @@ local function get( sUrl )
     response.close()
     return sResponse
 end
- 
+
 -- Determine file to download
 local sUrl = tArgs[1]
-local sFile = tArgs[2]
+
+--Check if the URL is valid
+local ok, err = http.checkURL( sUrl )
+if not ok then
+    if err then
```

Comment:
I wonder if it'd be worth doing `printError( err or "Failed. ")` instead. Though `err` shouldn't ever be nil, so neither is _really_ needed.

Reply:
If this is done it would probably be better to use "Invalid URL" or something like that instead of "Failed." to distinguish it from the "Failed." message caused when the download failed.